codewof/programming/content/en/driver-speed/question.md

```diff
@@ -0,0 +1,12 @@
+# Driver speed
+
+Write a function `check_speed(speed)` that checks if a driver is driving above the speed limit.
+
+- If the speed is less than or equal to 50, the function should **print** `Ok.`
+- If the speed is above 50, the function should **print** `Too fast!`
+- If the speed is 75 or higher then the function should *also* **print** `Licence suspended.`
+
+For example: `check_speed(53)` should **print** `Too fast!`, and
+`check_speed(70)` should **print**:  
+`Too fast!`  
+`Licence suspended.`
```

Comment:
This will (probably) stick `Too fast!` and `Licence suspended` on the same line, when we want them on different lines to match the expected result (sorry I missed this earlier)

I think it would work to use triple-backticks